packages/native/src/polywrap_native.rs

```diff
@@ -0,0 +1,27 @@
+use std::sync::Arc;
+
+use polywrap_client::core::{file_reader::SimpleFileReader, package::WrapPackage, wrapper::Wrapper};
+use polywrap_wasm::{wasm_package::WasmPackage, wasm_wrapper::WasmWrapper};
+
+use crate::{
+    error::FFIError,
+    package::FFIWrapPackage, wrapper::FFIWrapper,
+};
+
+pub fn ffi_wrap_package_from_bytecode(bytes: &[u8]) -> Result<Arc<FFIWrapPackage>, FFIError> {
```

Comment:
if we already have the `ffi_wrapper_from_bytecode` function do we really need this one?